data-products/_example/02-data-dictionary.yml

```diff
@@ -0,0 +1,79 @@
+---
+#-------------------------------------------------------------
+# Implementation notes
+#  - YAML validation required
+#  - Warn on missing recommended attributes
+#  - Warn on table & field names containing spaces or dashes
+#-------------------------------------------------------------
+databases:
+  # You must specify at least one database
+  prison_population:
+    # Add a description for your database to help users find it.
+    # This is optional, as the data product description will also aid discoverability.
+    description: "Extracted from the Offender Management Statistics Quarterly publication. Please refer to notes in that publication."
+    # Status: please specify beta, experimental, production
+    status: "production"
+    # If versioning is enabled, the Data Platform will store a historical snaphot of the database each time an update is sent.
+    # This can add SIGNIFICANTLY to storage costs, so you may be asked to justify this.
+    enable-versioning: false
+    tables:
```

Comment:
This implies that it's structural data, and I'd also suggest to use a term that is more domain oriented, instead of (database/table/column)

Reply:
This is a big one for debate. What's more familiar to users, vs. what behaviours do we want to encourage? For now I'll add an option to the data dictionary at the top - the user can say if they're using a tabular or domain model - that'll prompt us to hopefully support both in the future.